docs/cmake_doc/options.rst

```diff
@@ -320,8 +320,32 @@ NRN_ENABLE_TESTS:BOOL=OFF
 
   Clones the submodule catch2 from https://github.com/catchorg/Catch2.git and after a build using 
   ``make`` can run the tests with ``make test``.
-  May also need to ``pip install pytest``
-  If a test fails, how does one examine it in more detail?
+  May also need to ``pip install pytest``.
+  ``make test`` is quite terse. To get the same verbose output that is
+  seen with the travis-ci tests, use ``ctest -VV`` (executed in the
+  build folder). One can also run individual test files
+  with ``python3 -m pytest <testfile.py>`` or all the test files in that
+  folder with ``python3 -m pytest``. Note: It is helpful to ``make test``
+  first to ensure any mod files needed are available to the tests. If
+  running a test outside the folder where the test is located, it may be
+  necessary to add the folder to PYTHONPATH. Note: The last python
+  mentioned in the ``-DNRN_PYTHON_DYNAMIC=...`` (if the semicolon separated
+  list is non-empty and ``-DNRN_ENABLE_PYTHON_DYNAMIC=ON``)
+  is the one used for ``make test`` and ``ctest -VV``. Otherwise the
+  value specified by ``PYTHON_EXECUTABLE`` is used.
+
+  Example
+
+  .. code-block:: shell
+
+    mkdir build
+    cmake .. -DNRN_ENABLE_TESTS=ON ...
+    make -j
+    make test
+    ctest -VV
+    cd ../test/pynrn
+    python3 -m pytest
+    python3 -m pytest test_currents.py
```

Comment:
when we run tests via ctest (i.e. make test), internally we set PYTHONPATH as in https://github.com/neuronsimulator/nrn/blob/5feec9097814717d159225ddcdfb7bb8ab31f36a/test/CMakeLists.txt#L33

This is not required while running directly test via `python3 -m pytest test_currents.py` ?

Reply:
Good question. I thought about it but didn't know enough to come to a conclusion. The example finesses it as I have the ```python3 -m pytest``` implicitly launched within the folder containing the desired tests. The doc clearly needs another sentence about this. But why is it needed for rxd and not for the other folders?

Reply:
There is a sentence about it
 ``` If running a test outside the folder where the test is located, it may be necessary to add the folder to PYTHONPATH.```
Do you think that is adequate, or do you have a suggestion for enhancing the clarity? It would be nice to know the circumstances under which augmenting the PYTHONPATH is needed when running  a test outside the folder where the test is located.

Reply:
>  It would be nice to know the circumstances under which augmenting the PYTHONPATH is needed when running a test outside the folder where the test is located.

I have to check but if we launching python, isn't `PYTHONPATH` is always required for `neuron` module to be found?